### ISSUE TITLE
MultiScan cannot find scheme with test plans when container for scheme is a xcworkspace

### ISSUE THREAD

**Issue by vdavydovHH** [lyndsey-ferguson/fastlane-plugin-test_center#231]
### New Issue Checklist

- [x] Updated `fastlane-plugin-test_center` to the latest version
- [x] I read the [README.md](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/blob/master/README.md)
- [x] I reviewed the [example(s)](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/blob/master/README.md) for the action(s) I am using
- [x] I have removed any sensitive data such as passwords, authentication tokens, or anything else I do not want to world to see

### Issue Description
Hi @lyndsey-ferguson ! It's me again :)
I found a bug related to test plans. 
I get next message when container for my test plan is a .xcworkspace: `Error: cannot find any schemes in the Xcode workspace`

I attached example app where you could reproduce the problem.
With common `scan` all works fine.
Some useful commands to interact with project:

```
// Ruby Gems
bundle install
// Cocoapods
bundle exec pod install
// Test with multi_scan
bundle exec fastlane test_multi_scan
// Test with scan
bundle exec fastlane test_scan

```
[MultiScanTestPlansNew.zip](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/files/4763504/MultiScanTestPlansNew.zip)

If you need something else, please let me know.
Thank you in advance!

**Comment by lyndsey-ferguson** [comment]
Thanks, I have a crashing bug that will be of higher priority than this one. As a work-around, you can update your Fastfile to passing the `:only_testing` result of the new `test_options_from_testplan` action if you know the path of your testplan.

**Comment by lyndsey-ferguson** [comment]
Hello, @vdavydovHH! 

Can you modify your `Pluginfile` per my instructions below, run `bundle install`, and then run your fastlane again (with the `--verbose` flag)?

Pluginfile:
```
gem 'fastlane-plugin-test_center', :git => "https://github.com/lyndsey-ferguson/fastlane-plugin-test_center.git", :branch => "issue-231-tests-not-found-from-workspace"
```

If there are still problems, please let me know and attach the console output as a text file to this issue (makes it easier for me to review).  If it works, please let me know.

**Comment by lyndsey-ferguson** [comment]
And, thanks again for the sample projects, it really makes it easy to fix!

**Comment by vdavydovHH** [comment]
No problem!
It seems that everything all right 👍 

Could you release this fix and https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/issues/237 today?

**Comment by lyndsey-ferguson** [comment]
Fixed in [v3.11.4](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/releases/tag/v3.11.4)
